park_info.clj

```diff
@@ -1,198 +1,197 @@
-        ;---Basic Info---
-        :food true
-        :food_extra "café in Musaion museum of natural sciences (www.nm.cz)"
-        :toilet true
-        :toilet_extra nil
-        :POIs "Enjoyable location for a walk with children, Quiet and relaxing space in the center of town, Musaion Museum of Natural Sciences, decorative ponds, natural attraction - Petřín rock formations"
-        ;---Children and Sport---
-        :bike_path false
-        :skating false
-        :sports_field false
-        :sports_field_extra: nil
-        :playground true
-        :playground_extra: "yes, by the Kinských square entrance, nearby Kinských Villa and more under construction"
-        ;---Directions---
-        :public_transport "
-         stop: Kobrova - bus: #176"
-        :GPS_coords "50° 4′ 44.26″ N, 14° 24′ 0.21″ N"
-        :parking true
-        :parking_extra "OC Smíchov underground parking"
-        ;---Additional Information---
-        :path_type "paved"
-        :administered_by "Prague City"
-        :opening_hours "Open year-round"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/3/95/58/184325_4_kinskeho_mapa.pdf"       
-       
-    }
-    def Klamovka{
-        ;---Basic Info---
-        :food true
-        :food_extra "restaurant (www.zahradnirestaurace.cz) & club Klamovka (www.klubklamovka.cz)"
-        :toilet true
-        :toilet_extra "in the restaurant and club"
-        :POIs "Historically significant sights, art gallery, musical club with restaurant in garden, playground for small children"
-        ;---Children and Sport---
-        :bike_path true
-        :skating false
-        :sports_field true
-        :sports_field_extra: "Tennis courts and sports facilities in Sokolovna gymnasium"
-        :playground true
-        :playground_extra "by the garden restaurant"
-        ;---Directions---
-        :public_transport "
-        stop: Klamovka - buses: #149, #191, #217, - trams: #4, #7, #9, #10, #58, #59"
-        :GPS_coords "50°4′17,04″ N, 14°22′39,72″ E"
-        :parking false
-        :parking_extra nil
-        ;---Additional Information---
-        :dogs true
-        :path_type "paved & asphalt"
-        :administered_by "Municipality Prague 5"
-        :opening_hours "Open year-round"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/16/84/48/184296_4_klamovka_mapa.pdf"       
-       
-    }
-    def Ladronka{
+(def Obora_Hvezda{
+    ;---Basic Info---
+    :food true
+    :food_extra "During Summer Season on the premises of the Villa (www.pamatniknarodnihopisemnictvi.cz), tel:. +420 235 357 938,
+    or at Libocká street, restaurant U Holečků, www.uholecku.cz,
+    or at Libocká studna, tel. +420 777 836 966,
+    alternatively at Petřiny"
+    :toilet true
+    :toilet_extra "Portable toilets on the premises, during summer season toilets are also available in the villa"
+    :POIs "Suitable for tourists and families with children, heritage trees, nature trail, natural attractions"
```

Comment:
Again, wouldn't start with 'suitable for...'.

Reply:
These are literal translations from the website. Some parks have them, some don't. I Don't personally see a reason to change these.